backend/sqlmapper/drivers/pg_hook.go

```diff
@@ -31,33 +31,33 @@ func (s *pgHookStore) ColumnMetadata(q sqlmapper.Query) ([]database.Column, erro
 	return s.pgStore.ColumnMetadata(q)
 }
 
-func (s *pgHookStore) Create(tableName string, d sqlmapper.RowData) (sqlmapper.RowData, error) {
-	ctx := d.ToCtx()
+func (s *pgHookStore) Create(tableName string, row sqlmapper.RowData, relateRowData map[string][]sqlmapper.RowData) (sqlmapper.RowData, map[string][]sqlmapper.RowData, error) {
```

Comment:
we should not distinct relation_data and data and also we should not change interface